.github/workflows/ci.yml

```diff
@@ -94,9 +106,15 @@ jobs:
       - name: Build
         run: |
           cd ${GITHUB_WORKSPACE}/build
+          ninja search_family_test
```

Comment:
Why we need this?

Reply:
it's an arbitrary target that we would build anyway provides a consistent target being built. helps if there is a failure like disk space which gets lost when we build multiple targets in parallel.